### ISSUE TITLE
How to resolve registred types (without constructor)?

### ISSUE THREAD

**Issue by sjahongir** [NancyFx/Nancy#2810]
### Prerequisites

- [ ] I have written a descriptive issue title
- [ ] I have verified that I am running the latest version of Nancy
- [ ] I have verified if the problem exist in both `DEBUG` and `RELEASE` mode
- [ ] I have searched [open](https://github.com/NancyFx/Nancy/issues) and [closed](https://github.com/NancyFx/Nancy/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### System Configuration

- Nancy version:
- Nancy host
  - [ ] Nancy.Hosting.Aspnet
  - [x] Nancy.Hosting.Self
  - [x] Nancy.Owin ()
  - [ ] Other:
- Other Nancy packages and versions: v1.4.4
- Environment (Operating system, version and so on):
- .NET Framework version: 4.6.1
- Additional information:

Nancy.TinyIoc.TinyIoCResolutionException throwing, when I resolve `SyncService` as following:
`var service = TinyIoCContainer.Current.Resolve<SyncService>();`

How can I get instance of registred types from TinyIoCContainer?

**Comment by thecodejunkie** [comment]
@sjahongir What is the stack trace of the exception? Why are you accessing the container like that? It's a big anti-pattern and not something we really support. It would help if you explained why you need to do this, in order for us to provide a good answer

**Comment by sjahongir** [comment]
@thecodejunkie, I want to get instance of type in anywhere.

**Comment by thecodejunkie** [comment]
Yeah and that's the problem right there.. WHY do you need this. It sounds like there's an inherit design flaw if you need to do that.

**Comment by thecodejunkie** [comment]
tl;dr

If you want to use `TinyIoCContainer.Current` when you need to override the [GetApplicationContainer](https://github.com/NancyFx/Nancy/blob/master/src/Nancy/DefaultNancyBootstrapper.cs#L76) and return your own instance (in this case `TinyIoCContainer.Current`).

```
⚠️ You are probably doing something wrong if you need direct access to the container, from you code, especially if you need it "anywhere"
```

**Comment by sjahongir** [comment]
Thank you for your answer!
